### ISSUE TITLE
Write story page

### ISSUE THREAD

**Issue by kindacoder** [she-endeavors/SheE#9]
**Brief description of the feature? Please describe.**
develop write story page 
**Links and resources**
[https://adobe.ly/2uwTJkV](https://adobe.ly/2uwTJkV)
**Screenshots for the new feature**
[https://adobe.ly/2uwTJkV](https://adobe.ly/2uwTJkV)
**Additional Information**
Additional details about the feature

**Comment by devRD** [comment]
Is it only the frontend part?

**Comment by kindacoder** [comment]
@devRD No, we just have to focus on front-end part as of now. In this issue only this page needed to be developed

**Comment by devRD** [comment]
can I work on this?

**Comment by kindacoder** [comment]
yeah sure. Before that, we are running out of time, so it would be great, if we can discuss the timeline and work accordingly.let me know when you are going to create a PR for this one ?

**Comment by devRD** [comment]
If there's a time limit, I'll be sure to make a PR within the stipulated time. Ideally, if possible I can make a PR by tomorrow or at most the day after.

**Comment by kindacoder** [comment]
sure , so I'm assigning this one to you, make sure you make create PR on time :)

**Comment by devRD** [comment]
Will this be created in a different file or in index.html itself?

**Comment by kindacoder** [comment]
just clone the project and then start working :)

**Comment by devRD** [comment]
@kindacoder I sent you a message on slack yesterday, can you please check it out?

**Comment by kindacoder** [comment]
@devRD please share the screenshots

**Comment by palak2001** [comment]
@kindacoder can I work on it??

**Comment by kindacoder** [comment]
@palak2001 yeah
